.github/actions/create-release/gh-release-script.js

```diff
@@ -8,76 +8,77 @@ const myArgs = process.argv.slice(2);
-        exit(0); //skip this package but continue trying to release others
-    }
+  if (version.includes('-')){
+    exit(0); //skip this package but continue trying to release others
+  }
 }
 
 function gatherReleaseInfo(logPath) {
-    const changeLogs = fs.readFileSync(logPath, 'utf8');
-    const regex = /## \[([0-9]+(\.[0-9]+)+)] - [0-9]{4}-[0-9]{2}-[0-9]{2}/i;
-    
-    let lines = changeLogs.split(/\n/);
-    let foundChangelog = false;
-    let releaseInfo = '';
-    let i = 0;
+  const changeLogs = fs.readFileSync(logPath, 'utf8');
+  const regex = /## \[([0-9]+(\.[0-9]+)+)] - [0-9]{4}-[0-9]{2}-[0-9]{2}/i;
 
-    for(let j = 0; j < lines.length; j++){
-       if(lines[j].includes(`[${version}]`)){
-           i = j;
-           j = lines.length;
-           foundChangelog = true;
-       } 
-    }
+  let lines = changeLogs.split(/\n/);
+  let foundChangelog = false;
+  let releaseInfo = '';
+  let i = 0;
 
-    lines = lines.slice(i);
-
-    if(foundChangelog){
-        for(let j = 0; j < lines.length; j++){
-            if(j == 0){
-               releaseInfo += `${lines[j]}`+ '\n';
-               continue;
-            }        
-    
-            if(!regex.test(lines[j])){
-                releaseInfo += `${lines[j]}`+ '\n';
-            } else {
-                j = lines.length;
-            }
-        }
+  for(let j = 0; j < lines.length; j++){
+    if(lines[j].includes(`[${version}]`)){
+      i = j;
+      j = lines.length;
+      foundChangelog = true;
     }
-    
-    if(releaseInfo === ''){
-        core.setFailed("No release info found, either missing in changelog or changelog is formatted incorrectly")
+  }
+
+  lines = lines.slice(i);
+
+  if(foundChangelog){
+    for(let j = 0; j < lines.length; j++){
+      if(j == 0){
+        releaseInfo += `${lines[j]}`+ '\n';
+        continue;
+      }
+
+      if(!regex.test(lines[j])){
+        releaseInfo += `${lines[j]}`+ '\n';
+      } else {
+        j = lines.length;
+      }
     }
+  }
+
+  if(releaseInfo === ''){
+    core.setFailed("No release info found, either missing in changelog or changelog is formatted incorrectly")
+  }
 
-    console.log("Gathered release info...")  
-    return releaseInfo;
+  console.log("Gathered release info...")
+  return releaseInfo;
 }
 
 async function publishRelease(releaseInfo) {
 
-    await request('POST /repos/{owner}/{repo}/releases', {
-        headers: {
-            authorization: `token ${process.env.REPO_TOKEN}`,
-        },
-        owner: 'subquery',
-        name: `[${version}] ${repoName}`,
-        repo: 'subql-algorand',
-        tag_name: `${packageName[1]}/${version}`,
-        body: releaseInfo
-    }).catch( err => {
-        core.setFailed(err)
-    })
-
-    console.log("Release Created...")  
+  await request('POST /repos/{owner}/{repo}/releases', {
+    headers: {
+      authorization: `token ${process.env.REPO_TOKEN}`,
+    },
+    owner: 'subquery',
+    name: `[${version}] ${repoName}`,
+    repo: 'subql',
```

Comment:
Maybe we should change this script to remove specific details

```suggestion
    repo: 'subql-algorand',
```